packages/blis/meta.yaml

```diff
@@ -0,0 +1,19 @@
+package:
+  name: blis
+  version: 0.7.9 # Currently the newest version crashes explosion's thinc library: https://github.com/explosion/thinc/issues/771
+  top-level:
+    - blis
+source:
+  url: https://files.pythonhosted.org/packages/ff/a1/0ece7b0cceda6ca764ecba21e32f099cebff3448d88aa33ff0fce708d34b/blis-0.7.9.tar.gz
+  sha256: 29ef4c25007785a90ffc2f0ab3d3bd3b75cd2d7856a9a482b7d0dac8d511a09d
+about:
+  home: https://github.com/explosion/cython-blis
+  PyPI: https://pypi.org/project/blis
+  summary: The Blis BLAS-like linear algebra library, as a self-contained C-extension.
+  license: BSD
+requirements:
+  host:
+    - numpy
+build:
+  script: |
+    export BLIS_ARCH="generic"
```

Comment:
Could you put the sections in the following order:
```
package
source
build
requirements
test
about
```
here and in other added packages?